### ISSUE TITLE
Why validated but Overfitting?

### ISSUE THREAD

**Issue by QingZhou-YangHY** [QingZhou-YangHY/ML_HW#1]
I used a validation set, but my model still overfitted?

**Comment by QingZhou-YangHY** [comment]
原因：待选择的模型太多了,抽到的Validation set不好.说白了就是把Validation Set当成了Training Set,不是头一次见到,所以就把答案"背"了下来,然后Overfitting.
